widget/check.go

```diff
@@ -100,6 +100,30 @@ type Check struct {
 
 	focused bool
 	hovered bool
+
+	checkSource   binding.Bool
+	checkListener binding.DataListener
+}
+
+// Bind connects the specified data source to this Check.
+// The current value will be displayed and any changes in the data will cause the widget to update.
+// User interactions with this Check will set the value into the data source.
+//
+// Since: 2.0.0
+func (c *Check) Bind(data binding.Bool) {
```

Comment:
`Bind` should first call `Unbind` to avoid the widget getting updated by a previously added listener, see #804